src/dlgTriggerEditor.cpp

```diff
@@ -358,6 +365,36 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     saveAction->setStatusTip(tr("Saves the selected trigger, script, alias, etc, causing new changes to take effect - does not save to disk though..."));
     connect(saveAction, SIGNAL(triggered()), this, SLOT(slot_save_edit()));
 
+    QAction* copyAction = new QAction(tr("Copy"), this);
```

Comment:
@vadi2 wrote:
> There are no more new QActions 😉

So, _what's this, Scotch mist_?

Reply:
Oh yeah, you're right.

Reply:
Ah, they are only present on the context (typically accessed with a right mouse button click) menu for the `QTreeWidget`s in the LHS of the Editor window - I had visions of toolbar buttons on the top (but movable) toolbar in the Editor.

Of course, without hunting down the context menus of the `QTreeWidgets` (will have to do a `findChild` as there is no direct method to get them) and then setting those same menus (with a call to `QMenu::setTooTipsVisible(true)`) to show tool-tips the tool-tips will not ever be seen IMHO...